appservice/src/SiteClient.ts

```diff
@@ -150,12 +152,14 @@ export class SiteClient implements IAppSettingsClient {
     }
 
     public async stop(): Promise<void> {
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
Do these lint errors go away if we return the value? In general, this eslint rule is kind of annoying since we use this pattern pretty often.

Reply:
What we should really be doing is if/else here, not this. But yeah, we could return it too.